templates/repo/settings/secrets.tmpl

```diff
@@ -0,0 +1,60 @@
+<div class="ui container">
+		<h4 class="ui top attached header">
+			{{.locale.Tr "secrets.secrets"}}
+			<div class="ui right">
+				<div class="ui primary tiny show-panel button" data-panel="#add-secret-panel">{{.locale.Tr "secrets.creation"}}</div>
+			</div>
+		</h4>
+		<div class="ui attached segment">
+			<div class="{{if not .HasError}}hide {{end}}mb-4" id="add-secret-panel">
+				<form class="ui form" action="{{.Link}}?act=secret" method="post">
+					{{.CsrfTokenHtml}}
+					<div class="field">
+						{{.locale.Tr "secrets.description"}}
+					</div>
+					<div class="field {{if .Err_Title}}error{{end}}">
+						<label for="title">{{.locale.Tr "secrets.name"}}</label>
+						<input id="ssh-key-title" name="title" value="{{.title}}" autofocus required pattern="^[a-zA-Z_][a-zA-Z0-9_.-]*$">
```

Comment:
1. `id` appears copy pasted
2. the `label for` only works on `id`, not on `name`.